src/Directory.Build.props

```diff
@@ -2,79 +2,18 @@
 <Project>
   <PropertyGroup>
     <IsUnitTestProject>false</IsUnitTestProject>
```

Comment:
Maybe just default this in the parent? and remove all the sets except actual test projects?